index.html

```diff
@@ -5,9 +5,37 @@
         <link rel="icon" type="image/svg+xml" href="/jenkins.svg" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Jenkins infra-statistics</title>
+        <!-- Load necessary scripts for jio-navbar -->
+        <script type="module" src="https://unpkg.com/@jenkinsci/jenkins-io-components?module"></script>
+        <script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.6.0/webcomponents-loader.js"></script>
+        <script src="https://unpkg.com/lit@2.4.0/polyfill-support.js"></script>
+        <script
+            data="ionicons"
+            src="https://cdnjs.cloudflare.com/ajax/libs/ionicons/5.5.2/ionicons/ionicons.esm.js"
+            type="module"
+        ></script>
+        <script
+            data="ionicons"
+            nomodule=""
+            src="https://cdnjs.cloudflare.com/ajax/libs/ionicons/5.5.2/ionicons/ionicons.js"
+        ></script>
     </head>
     <body>
         <div id="root"></div>
         <script type="module" src="/src/main.tsx"></script>
     </body>
 </html>
+
+<!-- <!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="icon" type="image/svg+xml" href="/jenkins.svg" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Jenkins infra-statistics</title>
+    </head>
+    <body>
+        <div id="root"></div>
+        <script type="module" src="/src/main.tsx"></script>
+    </body>
+</html> -->
```

Comment:
```suggestion
```

Reply:
@lemeurherve  

Since this was fixed with #18 , how do I prevent stale code from appearing here going forward?

I am not 100% familiar with the git workflow yet. 

Thank you

Reply:
@shlomomdahan You may want to rebase your local branch against the latest `main` branch if these commented out lines have already been removed in another PR